gammapy/datasets/make.py

```diff
@@ -64,3 +71,121 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def make_test_observation_table(observatory, n_obs):
+    """Generate an observation table.
+
+    For the moment, only random observation tables are created.
+
+    Parameters
+    ----------
+    observatory : string
+    	name of the observatory
+    n_obs : integer
+    	number of observations for the obs table
+
+    Returns
+    -------
+    obs_table : `~gammapy.obs.ObservationTable`
+    	observation table
+    """
+    n_obs_start = 1
+
+    obs_table = ObservationTable()
+
+    # obs id
+    obs_id = np.arange(n_obs_start, n_obs_start + n_obs)
+    col_obs_id = Column(name='OBS_ID', data=obs_id)
+    obs_table.add_column(col_obs_id)
+
+    # on time
+    ontime = Quantity(30.*np.ones_like(col_obs_id.data), 'minute')
+    col_ontime = Column(name='ONTIME', data=ontime)
+    obs_table.add_column(col_ontime)
+
+    # livetime
+    livetime = Quantity(25.*np.ones_like(col_obs_id.data), 'minute')
+    col_livetime = Column(name='LIVETIME', data=livetime)
+    obs_table.add_column(col_livetime)
+
+    # TODO: add columns for coordinates:
+    #       alt az
+    #       date -> done
+    #       calculate pointing observation (ra dec), considering the observatory location, and alt az is at the middle of the observation
+    #       then convert to alt az, considering the observatory location
+
+    # TODO: is there a way to comment on the column names?!!!!!
```

Comment:
@mapazarr – The `astropy.table.Column` class has a description, and you can set it here.
The problem is that CSV and even FITS don't support it:
```
from astropy.table import Table, Column
t = Table()
c = Column(name='aaa', data=[1,2,3], unit='m/s', description='Triple a')
t.add_column(c)
t.write('/tmp/aaa.fits')
from astropy.io import fits
header = fits.getheader('/tmp/aaa.fits')
header = fits.getheader('/tmp/aaa.fits', ext=1)
print(repr(header))
```
shows that units are saved in FITS, but description is not:
```
TTYPE1  = 'aaa     '                                                            
TFORM1  = 'K       '                                                            
TUNIT1  = 'm s-1   '                                                            
```
Now sometimes FITS tables do contain description comments of header keywords and columns, but with Astropy table these are lost when you do:
```
Table.read('file.fits').write('file2.fits')
```
I don't know if / how this will improve in the future, for now I'd say it's OK if you don't add a column description here, but it could also be useful in case the Astropy Table / FITS integration improves in the future or if we decide to use a format that can store all the Astropy Table meta-data like [ECSV](http://astropy.readthedocs.org/en/latest/api/astropy.io.ascii.Ecsv.html).